vnext/Desktop/module.g.cpp

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "winrt/base.h"
+void* winrt_make_facebook_react_MemoryTracker();
+void* winrt_make_facebook_react_NativeLogEventSource();
+void* winrt_make_facebook_react_NativeTraceEventSource();
+
+bool __stdcall winrt_can_unload_now() noexcept
+{
+    if (winrt::get_module_lock())
+    {
+        return false;
+    }
+
+    winrt::clear_factory_cache();
```

Comment:
>clear_factory_cache(); [](start = 11, length = 22)

why is this needed here in getter function?

Reply:
I do not know. It is generated by C++/WinRT.